docs/book/getting-started/core-concepts.md

```diff
@@ -86,17 +84,21 @@ In order to benefit from the aforementioned core concepts to their fullest exten
 
 First, in order to utilize _stack components_ that are running remotely on a cloud infrastructure, you need to deploy a **ZenML Server**, so that it can communicate with these stack components and run your pipelines.
 
-![Visualization of the relationship between code and infrastructure.](../.gitbook/assets/04_architecture.png)
+![Visualization of the relationship between code and infrastructure.](../.gitbook/assets/04\_architecture.png)
+
+#### Server Deployment
+
+In order to benefit from the advantages of using a deployed ZenML server, you can either choose to use the **ZenML Cloud** which provides a control plane for you to create managed instances of ZenML servers, or deploy it in your self-hosted environment.
```

Comment:
link to cloud?

Reply:
i.e. instead of just mentioning it? URL link.

Reply:
Would you prefer linking to the cloud directly or corresponding docs page?

Reply:
I think to the docs page is fine? I leave it up to you.

Reply:
Linked it.